### ISSUE TITLE
ChaseTaskTest randomly failing

### ISSUE THREAD

**Issue by ThatTechedGuy** [UQdeco2800/2021-ext-studio-2#83]
The test fails at line 54 of the ChaseTaskTest.java file

 https://github.com/UQdeco2800/2021-ext-studio-2/runs/3462219647?check_suite_focus=true

Failed pipeline:
> Task :core:test

ChaseTaskTest > shouldMoveTowardsTarget() FAILED
    org.opentest4j.AssertionFailedError at ChaseTaskTest.java:54

185 tests completed, 1 failed

FAILURE: Build failed with an exception.

* What went wrong:

> Task :core:test FAILED
Execution failed for task ':core:test'.

**Comment by lisslulay** [comment]
ChaseTaskTest is the task used by the ghost entity at first, and it seems that it has not been changed and I did not get this error in the local test.
-team 8

**Comment by lisslulay** [comment]
This problem exists. I compared the ChaseTask.java in main with the files in the game engine, and there is no difference. So this error is because someone changed some other file.

Now there is no error message because someone commented out the line 54 in ChaseTaskTest.java

--- team 8

**Comment by ThatTechedGuy** [comment]
Yes, @lisslulay my team (Team 7) commented it out because it was failing randomly. The test suite sometimes passes in the local environment but the pipeline still fails everytime. Hence, we commented it out since the player is also not being cased anymore by NPCs.

**Comment by lisslulay** [comment]
Team 8 used ChaseTask in this sprint and resumed its testing. We tested it many times and it was amazing that he did not report an error. If it fails again, please contact us and we will try our best to resolve it. :)
